### DISCUSSION TITLE
Aligning axes on timeseries

### DISCUSSION THREAD

**Discussion by basking-in-the-sun2000** [grafana/grafana#37291] [Dashboards, Panels & Visualizations]
Not sure if this is the place for this. The new timeseries has a lot of neat features, thanks

Have two different axes on a timeseries panel. The old graph had an option to align the axes. Understand you can now have multiple axis with this panel, so this might make it harder. It looks weird having the 0 lines somehow misaligned (one curve has a small negative component). Read somewhere that you could tweak the max/min to match the 0 axis, however, the values for this curve aren't predetermined and can vary

However, any chance for this when you have only two axis (left and right)?

<img width="555" alt="Screen Shot 2021-07-28 at 01 10 53" src="https://user-images.githubusercontent.com/61602111/127274224-e01b9659-05b2-4aa1-a37a-f796abe602d6.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
